tests/integration/test_vulnerability_detector/test_scan_results/test_scan_providers_and_nvd_feeds.py

```diff
@@ -79,6 +82,14 @@ def mock_vulnerability_scan(request):
     """
     It allows to mock the vulnerability scan inserting custom packages, feeds and changing the host system
     """
+    global nvd_vulnerabilities_number, provider_vulnerabilities_number, total_vulnerabilities
+    if "RHEL" in request.param['target']:
+        nvd_vulnerabilities_number = 0
+        total_vulnerabilities = provider_vulnerabilities['vulnerabilities']
+    else:
+        nvd_vulnerabilities_number = len(nvd_vulnerabilities['vulnerabilities'])
+        total_vulnerabilities = nvd_vulnerabilities['vulnerabilities'] + provider_vulnerabilities['vulnerabilities']
+    provider_vulnerabilities_number = len(provider_vulnerabilities['vulnerabilities'])
```

Comment:
Same comment as above, I think you could add such information in` system_data` rather than using global variables